app/views/projects.html

```diff
@@ -8,52 +8,201 @@
+                  <!-- TODO (MarkDeMaria): Insert Star funcionality here
+                  <div>
+                    <i class="fa fa-star-o" aria-hidden="true"></i>
+                  </div>
+                  -->
+
+                  <div class="project-names col-xs-12 col-sm-5 pad-none">
+                    <div class="project-summary mar-right-none">
+                      <h1 class="project truncate mar-bottom-none pad-bottom-sm">
+                        <a class="tile-target" href="project/{{project.metadata.name}}">{{project.metadata.annotations["openshift.io/display-name"]}}</a> <!-- {{(project | uniqueDisplayName : projects)}} -->
+                        <span ng-if="project.status.phase != 'Active'" data-toggle="tooltip" title="This project has been marked for deletion." class="pficon pficon-warning-triangle-o" style="cursor: help; vertical-align: top; margin-left: 5px;"></span>
+                      </h1>
+                    </div>
+                    <div>
+                      <p>{{project.metadata.name}}</p>
+                    </div>
+                    <div class="visible-xs">
+                      <div class="muted" style="margin-top: -5px;" ng-if="project | description">
+                        <truncate-long-text content="project | description" limit="100" use-word-boundary="true"></truncate-long-text>
+                      </div>
+
+                      <div row>
+                        <!-- TODO (MarkDeMaria): insert app retrieval here
+                        <div class="project-created text-center">
+                          <p class="h1 mar-top-md mar-bottom-none">#</p>
+                          <p class="mar-top-none pad-top-sm">Apps</p>
+                        </div>
+                        -->
+
+                        <!-- TODO (MarkDeMaria): insert status retrieval here
+                        <div flex class="project-created text-center mar-top-xl">
+                          <span class="mar-md">
+                            <i class="danger fa fa-2x fa-exclamation-triangle text-warning"></i>
+                            <tt class="h1 pad-left-sm">#</tt>
+                          </span>
+                          <span>
+                            <i class="danger fa fa-2x fa-times-circle-o text-danger"></i>
+                            <tt class="h1 pad-left-sm">#</tt>
+                          </span>
+                        </div>
+                        -->
+
+                        <div class="project-created text-left mar-top-md">
+                          <span class="text-muted h2 mar-bottom-none mar-top-none pad-top-none pad-bottom-none">
+                            Created
+                            <relative-timestamp timestamp="project.metadata.creationTimestamp"></relative-timestamp>
+                          </span>
+                        </div>
+
+                        <span flex></span>
```

Comment:
nit: can prob make `<span flex></span>` into `<div flex></div>` here so its the same node as its siblings.